ios/FluentUI.Demo/FluentUI.Demo/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.8.6</string>
+	<string>1.8.7/string>
```

Comment:
Missing an angle brace here (it's causing the build failures)

Reply:
Thanks for the catch!